src/index.html

```diff
@@ -1,12 +1,311 @@
-<!doctype html>
+<!-- #region head -->
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./styles/main.scss" />
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Catalog</h1>
+    <div class="conteiner">
+      <!-- #region header -->
+      <header class="main">
+        <div class="main-content">
+          <a href="#" class="main-content__logo">
+            <img src="./images/logo.png" alt="img" />
+          </a>
+
+          <nav class="main-content__nav">
+            <ul class="main-content__list">
+              <li class="main-content__item">
+                <a href="/" class="main-content__link is-active"> Apple </a>
```

Comment:
Need to fix the classes everywhere
```suggestion
          <nav class="nav">
            <ul class="nav__list">
              <li class="nav__item">
                <a href="/" class="nav__link is-active">Apple</a>
```